wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -46,12 +46,47 @@ <h3 class="usa-alert__heading">Provisional Data Statement</h3>
     </div>
 {%- endmacro %}
 
-{% macro NetworkComponent(network_cd, extent) -%}
+{% macro NetworkComponent(network_cd, extent, collection, narrative) -%}
     <div class="wdfn-component" data-component="network" data-networkcd="{{ network_cd }}" data-extent="{{ extent }}">
-        <h3>Network Map:</h3>
         <p id="overload-map"></p>
         <div id="network-map"></div>
-        <h3>Link Table:</h3>
+        <div id="network-metadata" class="usa-prose">
+            {% if narrative %}
+                <div>{{ narrative|safe }}</div>
+            {% endif %}
+
+            {% if collection.properties.contactInformation.contactName and collection.properties.contactInformation.contactEmail %}
+            <label class="bold-label" for="contact-info">Network Contact</label><br>
+            <span id="contact-info">
+                <span>{{ collection.properties.contactInformation.contactName }}</span>&nbsp;&nbsp;&nbsp;
```

Comment:
I'm not a big fan of using &nbsp; to enforce space between elements. Consider setting margin attributes in the style sheet.